embedded-graphics/src/primitives/mod.rs

```diff
@@ -30,42 +26,59 @@ pub trait Primitive: Dimensions {
 /// convenient macro.
 ///
 /// ```rust
-/// use embedded_graphics::pixelcolor::Rgb565;
-/// use embedded_graphics::prelude::*;
-/// use embedded_graphics::{egcircle, primitives::Circle};
-/// use embedded_graphics::style::{PrimitiveStyle, Styled};
-///
-/// let line_circle: Styled<Circle, PrimitiveStyle<Rgb565>> = egcircle!((10, 20), 30);
-/// let line_circle: Styled<Circle, PrimitiveStyle<Rgb565>> = egcircle!(Point::new(10, 20), 30);
+/// use embedded_graphics::{
+///     egcircle,
+///     pixelcolor::Rgb565,
+///     prelude::*,
+///     primitive_style,
+///     primitives::Circle,
+///     style::{PrimitiveStyle, Styled},
+/// };
+///
+/// let line_circle: Styled<Circle, PrimitiveStyle<Rgb565>> =
+///     egcircle!(center = (10, 20), radius = 30);
+/// let line_circle: Styled<Circle, PrimitiveStyle<Rgb565>> =
+///     egcircle!(center = Point::new(10, 20), radius = 30);
```

Comment:
This should have a comment to explain that different ways of specifying points are possible.